src/openklant/tests/test_migrate.py

```diff
@@ -689,3 +693,74 @@ def test_rsin(self):
         )
 
         self.assertEqual(output, [f"{self.live_server_url}{partij_url}"])
+
+    def test_klant_with_telefoonnummer_creates_digitaaladres(self):
+        # Klant with telefoonnummer creates a new DigitaalAdres
+        klant_with_phone = Klant(telefoonnummer="0612345678")
+
+        subject = Partij(nummer="12345", soort_partij=SoortPartij.persoon)
+        klant_with_phone._get_subject = lambda: subject
+
+        with patch(
+            "openklant.management.commands.migrate_to_v2_phonenumbers.OpenKlantClient"
+        ) as MockClient:
+            mock_client = MockClient.return_value
+            mock_client.retrieve.return_value = [
+                {"uuid": str(subject.uuid), "digitale_adressen": []}
+            ]
```

Comment:
It's a bit unfortunate that for the `migrate_to_v2` tests we didn't really have a proper docker-compose setup with Open Klant v1, but I'd rather use VCR here than mocking stuff like this. Making this into a proper setup with VCR with a docker-compose will be annoying, because it's difficult to unittest different cases since we make calls to the list endpoint. Also this script is only a one time thing for Den Haag, so instead you could copy one of the existing cassettes and modify it to add a phonenumber

Also instead of calling `_save_klanten_phonenumbers`, could you make sure the tests are run the same as the `migrate_to_v2` tests above (by using `call_command`)?